lib/jekyll-crds/filters/media_url_filter.rb

```diff
@@ -0,0 +1,21 @@
+module Jekyll
+  module MediaUrlFilter
+
+    def media_url(item, arg = nil)
+      # binding.pry
+      if item.nil? 
+        return
+      end
+
+      item["bitmovin_url"].nil? ? item.url : append_video_params(item, arg)
+    end
+
+    def append_video_params(item, arg)
+      item.url << "?"
+      arg ? item.url << arg : item.url << "autoPlay=true&sound=11"
```

Comment:
Good grief. This is gnarly as hell. 

Perhaps, this would be a little more readable... 

```ruby
if arg.present?
  "${item.url}?#{arg}"
else
  "${item.url}?autoPlay=true&sound=11"
end
```

Reply:
done